trunk/NDHMS/Routing/module_GW_baseflow.F

```diff
@@ -216,8 +216,14 @@ subroutine simp_gw_buck_nhd(           &
 
 ! Assume exponential relation between z/zmax and Q...
 !DJG force asymptote to zero to prevent 'overdraft'... 
-               qout_gwsubbas(bas) = C(bas)*(EXP(ex(bas)*z_gwsubbas(bas)/(z_mx(bas)/1000.))-1) !Exp.model. q_out (m^3/s)
+               if(GWBASESWCRT.eq.1) then ! Exponential model
+                   qout_gwsubbas(bas) = C(bas)*(EXP(ex(bas)*z_gwsubbas(bas)/(z_mx(bas)/1000.))-1) ! q_out (m^3/s)
+
+               elseif(GWBASESWCRT.eq.4) then ! Updated exponential model normalized by area
+                   qout_gwsubbas(bas) = C(bas)*basns_area(bas)*1.E-6*(EXP(ex(bas)*z_gwsubbas(bas)/(z_mx(bas)/1000.))-1) ! q_out (m^3/s)
```

Comment:
I dont particularly like the magic number 1.E-6. I'm assuming that it's a km^2/m^2 conversion to convert basin area to meters. Would be more readable if it were a variable named to that effect.

Reply:
Perhaps more mysterious is the preexisting 1000.

Reply:
Yeah, both of these assumptions are true (km^2/m^2 and the units issue with respect to z).  

I can add these conversions in as variables quickly.  

I would have liked to have cleaned up this code a bit more, but was worried about timelines and wasn't sure if this was still on the docket to be refactored.